### PR TITLE
docs(README): add index

### PR DESCRIPTION
**Pull request by bhavya9107**
- [ ] This is a **bugfix**
- [ ] This is a **feature**
- [ ] This is a **code refactor**
- [ ] This is a **test update**
- [x] This is a **docs update**
- [ ] This is a **metadata update**

### Motivation / Use-Case
Better navigation

### CODE REVIEW

**Comment by evilebottnawi** [comment]
README already very small, no need navigation, i think

**Comment by bhavya9107** [comment]
> README already very small, no need navigation, i think

it makes doc better and more efficient to be referred. I think it must be there

File: README.md [hunk 1]
```diff
@@ -18,6 +18,15 @@ live reloading. This should be used for **development only**.
 
 It uses [webpack-dev-middleware][middleware-url] under the hood, which provides
 fast in-memory access to the webpack assets.
+## Table of Contents
```
Comment (evilebottnawi) [inline README.md]: Add newline before and after header

Comment (bhavya9107) [reply README.md]: DONEE

Comment (bhavya9107) [reply README.md]: please reply on this

**Comment by evilebottnawi** [comment]
Thanks!

**Pull request merged**